snapcraft.yaml

```diff
@@ -75,6 +75,10 @@ plugs:
    interface: content
    target: $SNAP/gpu-2404
 #   default-provider: mesa-2404
+ external-qemu:
```

Comment:
Looking at the other plugs, and the snap feature options on https://snapcraft.io/lxd it seems like we usually use `noun.adjective` format, so this would be more in keeping to be called `qemu-external` in my view.

Reply:
Thanks! Have changed that.